ui/selectors/confirm-transaction.js

```diff
@@ -153,19 +154,6 @@ export const txDataSelector = (state) => state.confirmTransaction.txData;
 const tokenDataSelector = (state) => state.confirmTransaction.tokenData;
 const tokenPropsSelector = (state) => state.confirmTransaction.tokenProps;
 
-const contractExchangeRatesSelector = (state) => {
-  const chainId = getCurrentChainId(state);
-  const contractMarketData = state.metamask.marketData?.[chainId];
-
-  return Object.entries(contractMarketData).reduce(
-    (acc, [address, marketData]) => {
-      acc[address] = marketData?.price ?? null;
-      return acc;
-    },
-    {},
-  );
-};
```

Comment:
This chunk was identical to the `getTokenExchangeRates` selector, except it did not include a `?? {}` to handle the undefined case.  So I just made it re-use that selector instead.